src/modules/roc_packet/target_posix/roc_packet/address.cpp

```diff
@@ -124,6 +154,29 @@ bool Address::get_ip(char* buf, size_t bufsz) const {
     return true;
 }
 
+bool Address::get_multicast_iface(char* buf, size_t bufsz) const {
+    switch (miface_family_) {
+    case AF_INET:
+        if (!inet_ntop(AF_INET, &miface_.addr4, buf, (socklen_t)bufsz)) {
+            return false;
+        }
+
+        break;
+
+    case AF_INET6:
+        if (!inet_ntop(AF_INET6, &miface_.addr6, buf, (socklen_t)bufsz)) {
+            return false;
+        }
+
+        break;
+
+    default:
+        return false;
+    }
+
+    return true;
```

Comment:
false?

Reply:
It is the success branch. If something goes wrong we will return false.